ibm/service/kms/resource_ibm_kms_key_rings.go

```diff
@@ -35,6 +36,13 @@ func ResourceIBMKmskeyRings() *schema.Resource {
 				Description:  "User defined unique ID for the key ring",
 				ValidateFunc: validate.InvokeValidator("ibm_kms_key_rings", "key_ring_id"),
 			},
+			"force_delete": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "set to true to force delete this key ring. This allows key ring deletion as long as all keys inside are in a delete state",
```

Comment:
Suggestion:

This allows key ring deletion as long as all keys inside have `status` equals to 5 (destroyed).

Reply:
Implemented new wording to this and to the example for delete key ring in docs
https://github.com/IBM-Cloud/terraform-provider-ibm/pull/4767/commits/4de7118adf5d0ac788052ee8bf6f719f20f5b2e5